src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/MulticlassLogisticRegression.cs

```diff
@@ -45,8 +45,11 @@ public sealed class MulticlassLogisticRegression : LbfgsTrainerBase<MulticlassLo
 
         public sealed class Options : OptionsBase
         {
-            [Argument(ArgumentType.AtMostOnce, HelpText = "Show statistics of training examples.", ShortName = "stat", SortOrder = 50)]
-            public bool ShowTrainingStats = false;
+            /// <summary>
+            /// If set to <value>true</value>training statistics will be generated at the end of training.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Show statistics of training examples.", ShortName = "stat, ShowTrainingStats", SortOrder = 50)]
+            public bool ShowTrainingStatistics = false;
```

Comment:
>ShowTrainingStatistics [](start = 24, length = 22)

Can you double check it's used?
It strange binary LR and MultiLR has different options (binary LR has extra thing `ComputeLogisticRegressionStandardDeviation`)